### PR TITLE
[FSSDK-11133] Add cmab to project config

### PR DESCRIPTION
**Pull request by Mat001**
User Story
- Update project config to track CMAB properties

Context
- Under the Project Config Update Section of CMAB

Acceptance Criteria
- Complete the update
- Verify testing, add any code coverage as needed

https://jira.sso.episerver.net/browse/FSSDK-11133

### CODE REVIEW

File: pkg/config/datafileprojectconfig/config_test.go [hunk 1]
```diff
@@ -568,3 +574,94 @@ func TestGetFlagVariationsMap(t *testing.T) {
...
+	if experiment0.Cmab != nil {
+		// Test attribute IDs
+		assert.Equal(t, 2, len(experiment0.Cmab.AttributeIds))
+		assert.Contains(t, experiment0.Cmab.AttributeIds, "808797688")
+		assert.Contains(t, experiment0.Cmab.AttributeIds, "808797689")
+
+		// Test traffic allocation as integer
+		assert.Equal(t, 5000, experiment0.Cmab.TrafficAllocation)
+	}
+}
```
Comment (raju-opti) [inline pkg/config/datafileprojectconfig/config_test.go:621]: maybe we can add another test to check that cmab field is nil in config when it is not present in datafile?

**Review by raju-opti — APPROVED** [review]
LGTM. Few non-critical suggestions
